cmd/api/src/api/v2/anonymize.go

```
@@ -0,0 +1,312 @@
+			response)
+		return
+	}
+
+	// Phase 3: Write anonymized values into the graph
+	if err := s.Graph.WriteTransaction(ctx, func(tx graph.Transaction) error {
+		for _, t := range translations {
+			node, err := tx.Nodes().Filter(
+				query.Equals(query.NodeID(), t.NodeID),
+			).First()
+			if err != nil {
+				slog.WarnContext(ctx, "Anonymize: could not fetch node for update",
```

golangci-lint (2.11.4)

[major] arguments should be put on separate lines

(sloglint)